### ISSUE TITLE
Deleting ligatures does not update components, resulting in errors

### ISSUE THREAD

**Issue by Astavie** [glyphr-studio/Glyphr-Studio-1#304]
If you delete a ligature comprised of components, the "glyphs that use this component" section will not be updated on said components. Afterwards, when you try to click on the component in the components chooser, it will spit out an error in the console: 
```
Uncaught TypeError: getGlyph(...).changed is not a function
    at Glyph.changed ((index):49)
    at markSelectedWorkItemAsChanged ((index):49)
    at selectComponent ((index):49)
    at HTMLDivElement.onclick ((index):1)
```
This issue persists even after a save/reload.

**Comment by mattlag** [comment]
Good catch, this is a big and I'll fix it.

We keep a list of where components are used, and edge cases like this occur where the list doesn't get updated with changes.

Thanks!

**Comment by mattlag** [comment]
fixed in 1.13.09 - shipping later today!
